index.html

```diff
@@ -26,20 +26,20 @@
         <nav class="menu">
             <!-- Desktop Version Menu --> 
             <ul class="menu-items">
-                <li class="menu-item portfolio">
+                <li class="menu-item portfolio" tabindex="0">
                     <p>Portfolio</p>
                 </li>
-                <li class="menu-item about">
+                <li class="menu-item about" tabindex="0">
                     <p>About</p>
                 </li>
-                <li class="menu-item contact">
+                <li class="menu-item contact" tabindex="0">
                     <p>Contact</p>
                 </li>
             </ul>
 
             <!-- Mobile Version Menu -->
             <button class="menu-btn active" type="button">
```

Comment:
Some buttons have poor color contrast in the project. Please use [this](https://www.aditus.io/) link to analyze and correct them accordingly.